libs/store/src/lib/directives/ngLet.ts

```diff
@@ -7,30 +7,53 @@ import {
   OnInit,
 } from '@angular/core';
 
-export class NgLetContext {
-  $implicit: any = null;
-  ngLet: any = null;
+export class NgLetContext<T> {
+  $implicit: T = null;
+  ngLet: T = null;
 }
 
 @Directive({
   selector: '[ngLet]',
 })
-export class NgLetDirective implements OnInit {
-  private _context = new NgLetContext();
+export class NgLetDirective<T> implements OnInit {
```

Comment:
Breaking changes should be avoided, please add default types:

* change `class NgLetContext<T>` to `class NgLetContext<T = any>`
* change `class NgLetDirective<T>` to `class NgLetDirective<T = any>`

Reply:
Done!